### PR TITLE
Create GitHub issue cloud run service

### PR DESCRIPTION
**Pull request by dekiel**
**Description**
Create Github Issue service creates an github issue using data provided in an event.
Service accept cloudevents format.
Issue body is rendered from template to support complex input data types.

Changes proposed in this pull request:

- Created Cloud Run service creating a github issue.
- Created IssueData types and method rendering body.
- Dockerfile for cloud run service.

### CODE REVIEW

File: development/gcp/cloud-run/create-github-issue/create-github-issue.go [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+	if !ok {
+		crhttp.WriteHTTPErrorResponse(w, http.StatusInternalServerError, logger, "failed create github issues, received non 2xx response code: status code %d", response.StatusCode)
+		return
+	}
+	logger.LogInfo("created github issue: %s", issue.GetHTMLURL())
+	msg.GithubIssueNumber = issue.Number
+	msg.GithubIssueURL = issue.HTMLURL
+	msg.GithubIssueOrg = github.String(githubOrg)
+	msg.GithubIssueRepo = github.String(githubRepo)
+	// TODO: Add setting GithubAssigne in msg
```
Comment (Sawthis) [inline development/gcp/cloud-run/create-github-issue/create-github-issue.go:178]: should this TODO comment stay and be merged into main branch?

Comment (dekiel) [reply development/gcp/cloud-run/create-github-issue/create-github-issue.go:178]: Yes, assigning a user to github issue will be implemented in future.

**Pull request merged**